### PR TITLE
chore: use well items to get cursor from endpoints [release]

### PR DESCRIPTION
**Pull request by drsphelps**
Added a new data class to deal with the new endpoints, and add in endpoints to get all the labels available in the service.

### CODE REVIEW

File: packages/wells/src/client/api/wellsApi.ts [hunk 1]
```diff
@@ -14,26 +16,53 @@ export class WellsAPI {
     this.project = project;
   }
 
-  /* eslint-disable */
-  public list = async (): Promise<Well[] | undefined> => {
+  // public set setCluster(project: String) {
+  //   this.cluster = project;
+  // }
```
Comment (Sollimann) [inline packages/wells/src/client/api/wellsApi.ts]: remove?

**Review by Sollimann — APPROVED** [review]
Looks great! :+1: :100:

File: packages/wells/src/client/api/wellsApi.ts (hunk 1 above)
Comment (drsphelps) [reply packages/wells/src/client/api/wellsApi.ts]: ah yeah!

File: packages/wells/src/__tests__/testUtils.ts [hunk 2]
```diff
@@ -3,14 +3,18 @@
...
-    appId: `JS WELLS SDK integration tests (${name})`,
-    baseUrl,
-  });
+  return new CogniteWellsClient(
+    {
+      appId: `JS WELLS SDK integration tests (${name})`,
+      baseUrl,
+    },
+    Cluster.PROD
+  );
```
Comment (Sollimann) [inline packages/wells/src/__tests__/testUtils.ts]: ps, I removed baseUrl from the CogniteWellsClient constructor. We're already setting that inside of the class itself

File: packages/wells/src/client/api/wellboresAPI.ts [hunk 3]
```diff
@@ -15,15 +16,26 @@ export class WellboresAPI {
...
+  private getPath(baseUrl: string): string {
+    if (this.project == undefined){
+      throw new HttpError(400, "The client project has not been set.", {})
+    }
+    if (this.cluster == undefined) {
+      throw new HttpError(400, "No cluster has been set.", {})
+    }
+    baseUrl = `/${this.project}${baseUrl}?env=${this.cluster}`
+    return baseUrl
+  }
```
Comment (Sollimann) [inline packages/wells/src/client/api/wellboresAPI.ts]: good idea :+1:

**Review by Sollimann — APPROVED** [review]
Good job! :+1:

**Pull request merged**